Makefile

```diff
@@ -15,7 +16,11 @@ LIBSECCOMP_COMMIT := release-2.3
 
 LDFLAGS := -ldflags '-X main.gitCommit=${GIT_COMMIT} -X main.buildInfo=${BUILD_INFO} -X main.cniVersion=${CNI_COMMIT}'
 
-all: buildah imgtype docs
+all: buildah imgtype docs static
+
+.PHONY: static
+static: *.go imagebuildah/*.go bind/*.go chroot/*.go cmd/buildah/*.go docker/*.go pkg/cli/*.go pkg/parse/*.go unshare/*.c unshare/*.go util/*.go
+	$(GO) build -ldflags '-X main.gitCommit=ba976ed3 -X main.buildInfo=1533330547 -X main.cniVersion=v0.6.0' -ldflags "-extldflags '-static'" -o buildah.static -tags "$(TAGS) $(SELINUXTAG) containers_image_ostree_stub containers_image_openpgp exclude_graphdriver_devicemapper" ./cmd/buildah
```

Comment:
Can we find a cleaner way to do this?  Directly specifying the values that we're also storing in variables is bound to create mismatches when we eventually forget to update both copies when making a change to one or the other.

Reply:
Suggestion on how?

Reply:
Well, comparing the output of `make -n buildah` and `make -n static`, I see three build tags being added and a second `-ldflags` argument being used.

Adding a variable, similar to $(TAGS) but with no default value, that gets incorporated into $(LDFLAGS), and making the value of the `-o` argument to the `go build` command a variable with a default value of `buildah` would make it possible for the `static` build target to do its work by recursively invoking `$(MAKE)` with `TAGS="$(TAGS) addedtags`, and specifying values for the new variables that we use for holding the additional LDFLAGS and the binary name.